File: src/model/document_type.py
from enum import Enum


class DocumentType(Enum):
    CIN = 0
    DRIVING_LICENCE = 1
    BANK_CARD = 2
    NONE = 3


def get_type_by_num(n: int) -> DocumentType:
    if n == 1:
        return DocumentType.CIN
    elif n == 2:
        return DocumentType.DRIVING_LICENCE
    elif n == 3:
        return DocumentType.BANK_CARD
    else:
        return DocumentType.NONE

File: src/model/document.type.ts
export enum DocumentType {
  CIN,
  DRIVING_LICENCE,
  BANK_CARD,
  NONE
}

export const getTypeByNum = (n: number): DocumentType => {
  if (n == 1) return DocumentType.CIN;
  else if (n == 2) return DocumentType.DRIVING_LICENCE;
  else if (n == 3) return DocumentType.BANK_CARD;
  else return DocumentType.NONE;
};